build/makefiles/provider.mk

```diff
@@ -103,14 +104,14 @@ $(dest_par): $(bin_target0) Makefile Cargo.toml
 # par-full adds all the other targets to the base par
 par-full: $(dest_par) $(bin_targets)
 	for target in $(par_targets); do \
-	    target_dest=target/$${target}/release/$(bin_name);  \
+			target_dest=target/$${target}/release/$(bin_name);  \
```

Comment:
Because Makefiles are dumb sometimes, this doesn't mess with these scripts right?

Reply:
it shouldn't but I'll undo this change anyway

Reply:
while I'm there it looks like clippy has an issue

Reply:
Alright @brooksmtownsend I resolved the spacing (and lint) thanks for taking a look